Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Scripts/HideAfterEnabled.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.UI
```

Comment:
Have you tried running the example scene on device?
If so, what was the FPS?

Reply:
yes. It runs with 59-60 fps
i can change this to leverage the Update Loop instead of using Coroutines if you have perf concerns/ introducing inefficient coding patterns.

Reply:
No I just noticed it was red in your screenshot and made me curious

Reply:
Ahh.. That was a side effect of Remote Desktop session :)

---
In reply to: [327394090](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6049#discussion_r327394090) [](ancestors = 327394090)